README.md

```diff
@@ -29,6 +29,8 @@ If you are using flakes to configure your system, add to your nixpkgs overlays a
             {
               nixpkgs.overlays = overlays;
             };
+          #replace the configuration code segment above with the following if running nixos unstable:
```

Comment:
Since 22.11 this is the way to do it, you should just replace the entire `configuration` part, replace it with `modules` and remove the comment.